### ISSUE TITLE
Possibly unneeded "group by"

### ISSUE THREAD

**Issue by dmitryd** [FluidTYPO3/flux#756]
I believe grouping by uid is not needed in the following viewhelper and just adds extra evaluation to the SQL query:

https://github.com/FluidTYPO3/flux/blob/development/Classes/ViewHelpers/Content/GetViewHelper.php#L104

If it is correct, could you tell why it is there (just for my self education)?

Produced SQL statement is:

```SQL
SELECT * FROM tt_content WHERE ((tx_flux_column = 'column0:27735') OR (tx_flux_parent = '27735' AND (tx_flux_column = 'column0' OR tx_flux_column = 'column0:27735'))) AND deleted = 0 AND hidden = 0 GROUP BY uid  ORDER BY sorting ASC LIMIT 0,99999;
```

**Comment by NamelessCoder** [comment]
Hi Dmitry,

I agree that the `GROUP BY` part of the query is unnecessary. I seem to recall that in this particular case it is a side effect that is there because passing an empty value instead of a column name will generate an invalid query if you *also* pass a limit parameter. This may not be the case or may have since been fixed in the core - I don't think anyone has recently tested this with an empty `group` parameter; the code *was* recently refactored (by me) but I simply added all parameters that before, were passed to `$GLOBALS['TYPO3_DB']` instead of our RecordService (which is mainly there to provide an easier API to fetch records with simple conditions and to better allow unit tests to mock expected queries and responses).

If you determine it's no longer needed or know a better way to create these queries which avoids the case I've described, I would be very happy to receive a pull request or info about how to best patch this! :)
